### PR TITLE
Adds missing dot

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Once again I sign (I still don't know if I have to ?):
Guillaume Gomez : guillaume1.gomez@gmail.com

### CODE REVIEW

File: src/doc/intro.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ Segmentation fault (core dumped)
 
 A crash! (Note that this is actually system-dependent. Because referring to an
 invalid reference is undefined behavior, the compiler can do anything,
-including the right thing!) Even though we compiled with flags to give us as
+including the right thing!). Even though we compiled with flags to give us as
```
Comment (steveklabnik) [inline src/doc/intro.md]: This is not needed. Note that on 214, "A crash!" has an exclamation mark to end the sentence, and here, inside the `()`s, there's also an exclamation mark to end the sentence.

**Comment by steveklabnik** [comment]
You do not need to sign off on the commits.

Thank you for opening these, even though I'm not accepting this one!

File: src/doc/intro.md (hunk 1 above)
Comment (GuillaumeGomez) [inline src/doc/intro.md]: Oh sorry. But then, why "Even" has a majuscule ?

Comment (GuillaumeGomez) [inline src/doc/intro.md]: Oh sorry ! You're totally right, my bad !

**Comment by GuillaumeGomez** [comment]
No problem! Thanks for your precision !

**Pull request closed without merging**